### ISSUE TITLE
In ORKStepViewController, stepResult startDate sometimes null

### ISSUE THREAD

**Issue by jdkizer9** [ResearchKit/ResearchKit#928]
At ORKStepViewController.m#271, the result property getter can be invoked before self.presentedDate is set at viewWillAppear(ORKStepViewController.m#146). This can lead to stepResult.startDate being null. Swift bindings believe that startDate is a Date value (i.e., not an optional), which can lead to exceptions. To reproduce, see https://github.com/cornelltech/sdl-rkx/commit/f0c552d1f5ae44c9f7be94abe82dd41eddfdaf73

**Comment by syoung-smallwisdom** [comment]
This has already been fixed in master.

**Comment by emersonf** [comment]
@syoung-smallwisdom thanks Shannon. When do you expect the fix to land in `stable`?

**Comment by kelseydedoshka-apple** [comment]
@jdkizer9 - Thanks to @syoung-smallwisdom for pointing out this has already been resolved in master. Ok to close this issue or are you still experiencing problems?

**Comment by syoung-smallwisdom** [comment]
@kelseydedoshka-apple 
@emersonf has asked when the fix will be pushed to the stable branch. Any word on that?

**Comment by umerkhan-apple** [comment]
@syoung-smallwisdom We will let the community know as soon as we can.

**Comment by emersonf** [comment]
I did some basic testing yesterday and the fix on `master` seems to be
working.

We've switched to using the `master` branch until the multi-month release
process for `stable` changes. Our dependencies need to include fixes for
known bugs, especially those that affect us.

In a perfect world, `stable` would either include bug fixes or `master`
would get released more often. But `master` seems to be the right pick at
this time.
